usr/src/lib/libfakekernel/common/kmem.c

```diff
@@ -65,6 +67,29 @@ kmem_zalloc(size_t size, int kmflags)
 	return (umem_zalloc(size, kmem2umem_flags(kmflags)));
 }
 
+/*
```

Comment:
How about adding a strext.c file for this?
(like in uts/common/os)

It might not have been obvious, but we tried to keep the functions organized in files of similar names as the "real" versions.

Reply:
Will do.